build-system/tasks/runtime-test/index.js

```diff
@@ -506,6 +512,12 @@ async function runTests() {
       message += '\n';
       console./*OK*/log('\n');
       log(message);
+    }).on('run_complete', (browsers, results) => {
```

Comment:
For integration tests on sauce labs, this will cause the status to be reported once per batch of browsers. To fix, maybe you can keep a running total, and report the status once and for all here? (Once for stable, and once for beta?)

/cc @estherkim 

https://github.com/ampproject/amphtml/blob/1c54451b82cb9048b566d33ea3ad99e3a820183f/build-system/tasks/runtime-test/index.js#L367-L407

Reply:
It wouldn't - `run_complete` is called once, at the end of _all_ test runs

Reply:
Yes, but there are multiple test runs, for different sets of browsers. See `runTestInBatchesWithBrowsers()` in the comment above.

Reply:
SG! :+1: